api/rosetta/data_integration_test.go

```diff
@@ -170,7 +170,7 @@ func defaultNetwork() identifier.Network {
 }
 
 // defaultCurrency returns the Currency spec common for all requests.
-// At the moment only get the FLOW tokens, perhaps in the future it will support multiple.
+// For now this only gets the FLOW tokens.
```

Comment:
It should be `Flow token`. `FLOW` is the symbol only, and `tokens` describes a countable amount, not the _definition_.